acquia/acquia_cms
# ACMS-000: Fixing border focus issue on menu links.

**Motivation**
Gold border around focused links causes link to grow larger than container.
Fixes #NNN

**Proposed changes**
Increase specificity of CSS. Apply "display: block" property only focusable items with a role property of "link".

**Alternatives considered**
Switching the display of link focus border flex properties. Would have had larger impact that would be hard to test for.

**Testing steps**
* Navigate to any page. Use the tab key to add focus to main nav links. Ensure that the link doesn't jump around.
* Navigate to /search page and search for term that yields results. Tab through facets and ensure that gold border wraps focusable links. 

**Merge requirements**
- [ ] __Bug_
